curriculum/challenges/english/25-front-end-development/quiz-computer-basics/66ed8fb9f45ce3ece4053eac.md

```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
 
 #### --distractors--
 
-Placeholder distractor 1
+Processor
 
 ---
 
-Placeholder distractor 2
+Graphics Card
 
 ---
 
-Placeholder distractor 3
+Memory
 
 #### --answer--
 
-Placeholder answer
+Motherboard
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which is not a conventional way to connect a keyboard to the computer?
 
 #### --distractors--
 
-Placeholder distractor 1
+USB Receiver
 
 ---
 
-Placeholder distractor 2
+Bluetooth
 
 ---
 
-Placeholder distractor 3
+USB Cable
 
 #### --answer--
 
-Placeholder answer
+Ethernet Cable
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of the following ISP connections are typically the fastest?
 
 #### --distractors--
 
-Placeholder distractor 1
+DSL
 
 ---
 
-Placeholder distractor 2
+Satellite
 
 ---
 
-Placeholder distractor 3
+Cable
 
 #### --answer--
 
-Placeholder answer
+Fiber Optic
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of the following is NOT a safe practice when signing in?
 
 #### --distractors--
 
-Placeholder distractor 1
+Using a fingerprint or face scanner
 
 ---
 
-Placeholder distractor 2
+Making a long and complex password like `@ppl3_+B@nana34823`
 
 ---
 
-Placeholder distractor 3
+Utilizing a physical USB security key
 
 #### --answer--
 
-Placeholder answer
+Saving passwords in the browser on a public computer
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of the following is not a Web Browser?
 
 #### --distractors--
 
-Placeholder distractor 1
+Safari
 
 ---
 
-Placeholder distractor 2
+Opera GX
 
 ---
 
-Placeholder distractor 3
+DuckDuckGo
 
 #### --answer--
 
-Placeholder answer
+Google Drive
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+When you need to edit a piece of code, which of the following are you most likely going to use?
 
 #### --distractors--
 
-Placeholder distractor 1
+Node Package Manager
 
 ---
 
-Placeholder distractor 2
+Microsoft Edge
 
 ---
 
-Placeholder distractor 3
+Github
 
 #### --answer--
 
-Placeholder answer
+Visual Studio Code
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of the following stores and accesses files on the hard drive?
 
 #### --distractors--
 
-Placeholder distractor 1
+Dropbox
 
 ---
 
-Placeholder distractor 2
+Microsoft OneDrive
 
 ---
 
-Placeholder distractor 3
+Google Drive
 
 #### --answer--
 
-Placeholder answer
+Windows File Explorer
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is something you should avoid when naming files?
 
 #### --distractors--
 
-Placeholder distractor 1
+Putting an underscore or hyphen between words
 
 ---
 
-Placeholder distractor 2
+Use a casing style such as camelCase and sticking with it
 
 ---
 
-Placeholder distractor 3
+Putting a version number at the end of a document like `summaryReport_v2.docx`
 
 #### --answer--
 
-Placeholder answer
+Putting spaces between words
```

Comment:
It looks like question might be missing some part that would justify the answer. Otherwise having files with spaces between words is perfectly fine...

Reply:
oh

Reply:
Yes, there are situations, when spaces might cause some inconvenience (though none of the example would be a deal-breaker). Question doesn't mention any such context.

Reply:
ah ok, should I change it then?

Reply:
Yes please. Either just the answer, or whole question if you have idea for another one.

Reply:
Ok, I changed the answer.